libnavigation-core/src/test/java/com/mapbox/navigation/core/arrival/ArrivalProgressObserverTest.kt

```diff
@@ -305,9 +305,7 @@ internal class ArrivalProgressObserverTest {
             }
         }
         every { tripSession.getRouteProgress() } returns routeProgress
-        every { tripSession.updateLegIndex(2) } returns mockk {
-            every { legIndex } returns 2
-        }
+        every { tripSession.updateLegIndex(2) } returns true
```

Comment:
@kmadsen could you check that test changes are correct / covering all scenarios?

Reply:
LGTM